src/headline_grabber/__main__.py

```diff
@@ -30,7 +30,16 @@
     callback=validate_site_name,
     help="Comma-separated list of news sources to exclude from the search",
 )
-def main(include: str, exclude: str):
+@click.option(
+    "--entries",
+    "-n",
+    type=str,
+    default=None,
+    required=False,
+    callback=validate_max_entries,
+    help="Enter a number to specify the maximum number of entries",
```

Comment:
The naming and description of this option might be a bit misleading. The functionality places a limit on the results on a per section basis. The description should state something like: "Number specifying the maximum number of entries per section in a report"

Reply:
The description was changed.